src/pdm/cli/completions/pdm.zsh

```diff
@@ -2,6 +2,7 @@
 
 PDM_PYTHON="%{python_executable}"
 PDM_PIP_INDEXES=($(command ${PDM_PYTHON} -m pdm config pypi.url))
+PDM_CHECK_UPDATE=0
```

Comment:
Don't we need to `export`  this?

Reply:
Added the `export` and moved it before the `PDM_PIP_INDEXES` command (which is calling pdm)